project/Dependencies.scala

```diff
@@ -29,32 +29,29 @@ object Dependencies {
   }
 
   val googleDirectoryApiDependencies = Seq(
-    "com.google.api-client" % "google-api-client" % "1.28.0",
-    "com.google.apis" % "google-api-services-admin" % "directory_v1-rev32-1.16.0-rc"
+    "com.google.auth" % "google-auth-library-oauth2-http" % "0.20.0",
+    "com.google.apis" % "google-api-services-admin-directory" % "directory_v1-rev118-1.25.0"
```

Comment:
There's still a library eviction warning for Guava since this brings v20 and everything else is on v26 or v28. Need to test it still works fine

Reply:
This has been joined by another warning after I bumped Play 2.8:

```
[warn] 	* org.scala-lang.modules:scala-java8-compat_2.12:0.9.1 is selected over {0.8.0, 0.9.0}
[warn] 	    +- com.typesafe.play:play_2.12:2.8.2                  (depends on 0.9.1)
[warn] 	    +- com.typesafe.play:play-streams_2.12:2.8.2          (depends on 0.9.1)
[warn] 	    +- com.typesafe.play:play-ahc-ws-standalone_2.12:2.1.2 (depends on 0.9.0)
[warn] 	    +- com.typesafe.akka:akka-actor_2.12:2.6.5            (depends on 0.8.0)
```

That version of Akka comes transitively from Play itself. Sigh